Finished Test C19 on Issue #962

S-in-PNW opened the pull request:
# Description

Implements the test case C19 in [this document](https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md) into src/server/test/web/readingsCompareMeterFlow.js.

Assigned as a Codeday group project.
Group Members: @braedonbillingsley @Jinanmeotaku @S-in-PNW 
Mentor: @phvash

Partly Addresses #962

## Type of change

- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

## Limitations
N/A

S-in-PNW committed:
Completed test case C19

Co-authored-by: Jinan Almaliky<Jinanmeotaku@users.noreply.github.com> and Braedon Billingsley<braedonbillingsley@users.noreply.github.com>

S-in-PNW committed:
remove redundant comments

Chocopepero commented:
Hello, apologies for the delay on beginning the review. Before we get started, can all contributors sign the CLA? I don't see any of the contributors signatures in our records. Please let me know if this is an error.

braedonbillingsley commented:
> Hello, apologies for the delay on beginning the review. Before we get started, can all contributors sign the CLA? I don't see any of the contributors signatures in our records. Please let me know if this is an error.

Hello, we have all signed the CLA as of this moment. Please let us know if there's any other problems.

Chocopepero reviewed (approved):
The code looks good and the test runs as expected.

huss committed:
fix C19 description

This changes kWh to kW per the updated design
doc information. Until very recently it had the wrong
info. This was my fault and not the developers
who did this work.

huss reviewed (approved):
Per the comment by @Chocopepero, this work is fine. Congratulations to  @braedonbillingsley, @Jinanmeotaku & @S-in-PNW along with their mentor @phvash for their first accepted contribution to OED.
